### PR TITLE
[Example] ml service API name is changed

### PR DESCRIPTION
**Pull request by songgot**
- Add setting and start MQTT to md
- API name is changed from ml_remote_service_* to ml_service_remote_*
- bugfix: remove double free, unnecessary free function registered in function ml_option_set

**Self evaluation:**
1. Build test: [*]Passed [ ]Failed [*]Skipped
2. Run test: [*]Passed [ ]Failed [* ]Skipped

### CODE REVIEW

File: Tizen.native/MNIST_inference_after_training_offloading/README.md [hunk 1]
```diff
@@ -145,6 +145,18 @@ Example of inference of the number 9
 * Update Tizen package manager to the latest.
 * Prepare 2 RPI4 units.
 * RPI4 flashed with the latest tizen headed image.
+* Install mosquitto package on RPI4 to use MQTT, download rpm (mosquitto and mosquitto-client) from [this page](http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified/reference/repos/standard/packages/armv7l/)
+```
+  [host]
+  sdb push mosquitto-1.6.8-0.armv7l.rpm mosquitto-clients-1.6.8-0.armv7l.rpm
+  
+  [taget]
+  mount -o remount rw, /
```
Comment (gichan-jang) [inline Tizen.native/MNIST_inference_after_training_offloading/README.md:154]: I think this line is an overly kind guide.

Comment (songgot) [reply Tizen.native/MNIST_inference_after_training_offloading/README.md:154]: Okay, I will remove this line. Thank you.

**Review by gichan-jang — APPROVED** [review]
LGTM

File: Tizen.native/MNIST_inference_after_training_offloading/README.md [hunk 2]
```diff
@@ -145,6 +145,17 @@ Example of inference of the number 9
 * Update Tizen package manager to the latest.
 * Prepare 2 RPI4 units.
 * RPI4 flashed with the latest tizen headed image.
+* Install mosquitto package on RPI4 to use MQTT, download rpm (mosquitto and mosquitto-client) from [this page](http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified/reference/repos/standard/packages/armv7l/)
+```
+  [host]
+  sdb push mosquitto-1.6.8-0.armv7l.rpm mosquitto-clients-1.6.8-0.armv7l.rpm
+  
+  [taget]
+  rpm -Uvh --force --nodeps *.rpm
```
Comment (myungjoo) [inline Tizen.native/MNIST_inference_after_training_offloading/README.md:154]: 1. Unnecessary (and dangerous) options? --force, --nodeps
2. Add a note about the arch (armv7l vs aarch64). We have images of both architectures for RPI4.
3. taget --> target

File: Tizen.native/MNIST_inference_after_training_offloading/sender/src/sender.c [hunk 3]
```diff
@@ -567,33 +563,44 @@ start_model_receiver (appdata_s * ad)
...
+
+  /** A path to save the received model file */
+  g_autofree gchar *shared_path = app_get_shared_data_path ();
+  dlog_print (DLOG_ERROR, LOG_TAG, "path (%s)", shared_path);
+
+//  ret = ml_option_set (ad->option_h, "path", shared_path, NULL);
+//  if (ML_ERROR_NONE != ret) {
+//    dlog_print (DLOG_ERROR, LOG_TAG, "Failed to set path (%d)", ret);
+//    return;
+//  }
```
Comment (myungjoo) [inline Tizen.native/MNIST_inference_after_training_offloading/sender/src/sender.c:575]: Why commented this?
(it usually confuse other developers, especially if this is a sample app)

Comment (songgot) [reply Tizen.native/MNIST_inference_after_training_offloading/sender/src/sender.c:575]: I tried modifying rootstrap, but it didn't test well. so, this has not yet been verified.
I will conduct testing after the related PR( (https://github.com/nnstreamer/api/pull/411) is merged into Tizen.

File: Tizen.native/MNIST_inference_after_training_offloading/README.md (hunk 2 above)
Comment (songgot) [reply Tizen.native/MNIST_inference_after_training_offloading/README.md:154]: Thank you.

File: Tizen.native/MNIST_inference_after_training_offloading/sender/src/sender.c (hunk 3 above)
Comment (songgot) [reply Tizen.native/MNIST_inference_after_training_offloading/sender/src/sender.c]: Thank you. I will add a comment.

**Pull request merged**